docker-compose.yaml

```diff
@@ -49,7 +49,14 @@ services:
       - "8265:8265"
       - "10001:10001"
     # https://docs.ray.io/en/releases-2.30.0/cluster/cli.html#ray-start for more info about the command
-    command: bash -c "ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --ray-client-server-port 10001 --block" # pragma: allowlist secret
+    entrypoint:
+      - /bin/bash
+      - -c
+      - |
+          ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --ray-client-server-port 10001
+          mkdir -p /tmp/ray/session_latest/runtime_resources/pip
```

Comment:
When the Ray container is up and running it will create the `/tmp/ray/session_latest/runtime_resources/pip` directory, won't it? So, why do we need the `mkdir` command? If I'm correct, we can simplify this to:

```
ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --ray-client-server-port 10001
rmdir /tmp/ray/session_latest/runtime_resources/pip
ln -s /tmp/ray_pip_cache /tmp/ray/session_latest/runtime_resources/pip
sleep infinity
```

Also, what happens if `/tmp/ray/session_latest/runtime_resources/pip` is not empty? Would it make more sense to run `rm -rf /tmp/ray/session_latest/runtime_resources/pip` instead of using `rmdir`?

Reply:
Should we also add a `make` target to clean up the pip cache locally?

Reply:
Re: the first comment, you're probably right. Let me recheck the process.

Re: the make target, we have talked about [using labels](https://docs.docker.com/engine/manage-resources/labels/) to specify what to remove. Let's address this issue then.

Reply:
Oh, I found out (again) why the mkdir is needed :D The mount is only used for dev setups (local-up), so when starting a "standalone" setup this directory needs to be created. The -p ensures it won't complain because of the directory existing already.